hw/ip/padctrl/data/padctrl_fpv_testplan.hjson

```diff
@@ -0,0 +1,167 @@
+            valid write to the dio_pads register or the write enable signal raises.'''
+      milestone: V2
+      tests: ["padctrl_assert"]
+    }
+    {
+      name: Clk_A
+      desc: "Check that clk_pad_i is correctly connected to clk_o."
+      milestone: V2
+      tests: ["padring_assert"]
+    }
+    {
+      name: Rstpad_A
+      desc: "Check that rst_pad_ni is correctly connected to rst_no."
+      milestone: V2
+      tests: ["padring_assert"]
+    }
+    {
+      name: MioIn_A
+      desc: '''Check the muxed IO's output mio_in_o based on mio_pad_io and mio_attr_i inversion
+            bit.'''
+      milestone: V2
+      tests: ["padring_assert"]
+    }
+    {
+      name: MioInBackwardCheck_A
+      desc: '''If muxed IO's output mio_in_o has changed, then mio_pad_io or mio_attr_i must be
+            changed in the same clock cycle.'''
+      milestone: V2
+      tests: ["padring_assert"]
+    }
+    {
+      name: MioOutNormal_A
+      desc: '''If the selected muxed IO is enabled and the corresponding attribute is not open
+            drain, then the mio_pad_io should be the calculated value.'''
+      milestone: V2
+      tests: ["padring_assert"]
+    }
+    {
+      name: MioOutOd0_A
+      desc: '''If the selected muxed IO is enabled and the open drain attribute is set, if the
+            expected mio_output value is 0, then the mio_pad_io should be 0.'''
+      milestone: V2
+      tests: ["padring_assert"]
+    }
+    {
+      name: MioOutOd1_A
+      desc: '''If the selected muxed IO is enabled and the open drain attribute is set, if the
+            expected mio_output value is not 0, then the mio_pad_io could be either 0, 1, x or
+            high z.'''
+      milestone: V2
+      tests: ["padring_assert"]
+    }
+    {
+      name: MioOutBackwardCheck_A
+      desc: '''If muxed IO's output mio_pad_io has changed and mio_oe_i is enabled, then mio_attr_i
+            or mio_output_value must be changed in the same clock cycle.'''
+      milestone: V2
+      tests: ["padring_assert"]
+    }
+    {
+      name: MioOe_A
+      desc: '''If the selected muxed IO is not enabled, the mio_pad_io could be either 0, 1, x or
+            high z.'''
+      milestone: V2
+      tests: ["padring_assert"]
+    }
+    {
+      name: DioIn_A
+      desc: '''Check the dedicated IO's output dio_in_o based on dio_pad_io and dio_attr_i
+            inversion bit.'''
+      milestone: V2
+      tests: ["padring_assert"]
+    }
+    {
+      name: DioInBackwardCheck_A
+      desc: '''If dedicated IO's output dio_in_o has changed, then dio_pad_io or dio_attr_i must be
+            changed in the same clock cycle.'''
+      milestone: V2
+      tests: ["padring_assert"]
+    }
+    {
+      name: DioOutNormal_A
+      desc: '''If the selected dedicated IO is enabled and the corresponding attribute is not open
+            drain, then the dio_pad_io should be dio_out_i value XNOR the inversion bit.'''
```

Comment:
I think this is an XOR not XNOR... can you make sure the DIO/MIO descriptions are equal? looks like this description is different from `MioOutNormal_A`...